docs/flow_filtering.md

```diff
@@ -0,0 +1,95 @@
+# eBPF Flow Rule Based Filtering
+
+## Introduction 
+
+Flow rule-base filtering is a method to control the flow of packets cached in eBPF's flows table based on certain configuration
+
+## Flow filter rule configuration
+
+ The Flow filter rule consists of two parts mandatory and optional parameters.
+ 
+### Mandatory parameters
+
+- `FLOW_FILTER_IP_CIDR` - IP address and CIDR mask for the flow filter rule, supports IPv4 and IPv6 address format.
+  If wanted to match against any IP, user can use `0.0.0.0/0` or `::/0` for IPv4 and IPv6 respectively.
+- `FLOW_FILTER_ACTION` - Action to be taken for the flow filter rule. Possible values are `Accept` and `Reject`.
+  - For the matching rule with `Accept` action this flow will be allowed to be cached in eBPF table, with updated global metric `FlowFilterAcceptCounter`.
+  - For the matching rule with `Reject` action this flow will be dropped and not cached in eBPF table, with updated global metric `FlowFilterRejectCounter`.
+  - If the rule is not matched, the flow will be allowed to be cached in eBPF table, with updated global metric `FlowFilterNoMatchCounter`.
```

Comment:
this is also implicit `Accept`, no? I realize `FlowFilterAcceptCounter` and `FlowFilterRejectCounter` are specific if filter is matched, as opposed to having a metric, I wonder if there a better way to flag when the filter is matched vs not matched may be propagating to our consolePlugin ?

Reply:
we have 3 counters one for accept action and another for reject and the last one for nomatch flows
can u explain why nomatch metrics isn't good enough also this can be used with/without console

Reply:
it's fine to have NoMatch filter, I just thought NoMatch would be equivalent of "Accept" but I see your point with help of this metric we could identify the filter is being applied or not. thanks!